src/index.html

```diff
@@ -11,12 +11,67 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-logo">
+        <a href="#">
+          <img
+            src="images/logo.png"
```

Comment:
Consider providing a more descriptive `alt` attribute for the logo image. This will improve accessibility for users who rely on screen readers.